Lurk/Evaluation/Expr.lean

```diff
@@ -114,7 +114,7 @@ where
   telescopeLamAux (expr : Expr) (bindAcc : Array String) :=
     match expr with
     | .lambda name body => telescopeLamAux body $ bindAcc.push name
-    | _ => (expr, bindAcc)
+    | _ => (expr, bindAcc.reverse)
```

Comment:
If you're needing to reverse, then you probably want to use a `List` and stack in the beginning